README.md

```diff
@@ -1,15 +1,136 @@
+- [Other Documentation for Developers](#other-documentation-for-developers)
+ - [Adding dependencies via yarn](#adding-dependencies-via-yarn)
+ - [Updating translation export via yarn](#updating-translation-export-via-yarn)
+ - [Static build scripts](#static-build-scripts)
+ - [Sass Code Style Guidelines](#sass-code-style-guidelines)
+ - [Accessibility Guidelines](#accessibility-guidelines)
+
+## Getting Started
+
+In order to run Janis locally you need to [install Docker](https://docs.docker.com/install/#supported-platforms). Our team uses Docker for Mac.
+
+Once Docker is installed and you've cloned the code from this repository, you can run the "serve-local" Shell script below to spin up a Docker container and install dependencies within a virtualized environment that replicates the production environment in our Heroku deployment.
+
+#### As a React App (Developer Mode):
 ```
 ./scripts/serve-local.sh
 ```
 Your react app should be running at http://localhost:3000/
 
-## Adding dependencies via yarn
+**As a static build**
+To run the site locally as a static build, the way it works in production, see the [Static build scripts](#static-build-scripts) section below.
+
+
+## Project Info
+
+This work began under the he Austin Digital Services Discovery Project. Design, Technology, and Innovation Fellows worked in partnership between two City departments, Communications Public Information and Communications and Technology Management.
+
+We set out to learn what’s working for other government service providers, then embark on an iterative research, prototyping, and testing process to identify new designs, methods, and technologies.
+
+More high level information about the project from content strategy, user research and design, and development teams is continually updated on [our project page](http://projects.austintexas.io/projects/austin-digital-services-discovery/about/what-we-are-doing/).
+
+## Design
+
+Current design mockups are stored in Google Drive as Sketch Files:
+https://drive.google.com/drive/u/1/folders/1Xg6739ixhIDrb7MdJAnO4_zCRu3kDmZQ
+
+
+## Frameworks and Libraries Used
+
+Here are some high-level notes about the open source technologies we are using for this project.
+
+For more information about the concepts and architectural decisions guiding this work, take a break from this README and check out some of what we have written on Medium and our project page:
+
+- https://medium.com/civiqueso/open-source-city-cms-part-1-607a58b32356
+- http://projects.austintexas.io/projects/austin-digital-services-discovery/about/Dev/
+- http://projects.austintexas.io/projects/austin-digital-services-discovery/our-technical-approach/open-by-default/
+
+### React-Static
+
+This project uses [React-Static](https://github.com/nozzle/react-static) as a base framework for building static-progressive a React application/website. It's designed to with considerations for SEO, site performance, and user/developer experience.
```

Comment:
`for building static-progressive a React application/website`

`It's designed to with considerations`